### ISSUE TITLE
Flatpak: Attempting to enable 'Auto Install/Update DXVK-NVAPI on Prefix' leads to endless 'Please wait...' loop

### ISSUE THREAD

**Issue by helpvisa** [Heroic-Games-Launcher/HeroicGamesLauncher#3334]
### Describe the bug

Attempting to enable the DXVK-NVAPI auto-install process in a given prefix / game leads to an endless loop of the 'Please wait..' text animation and nothing happening. True of games installed in prefixes located in the home directory, as well as external drives.

### Add logs

```text
(17:50:21) INFO:    [Legendary]:        Legendary location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(17:50:21) INFO:    [Gog]:              GOGDL location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(17:50:21) INFO:    [Connection]:       Connectivity: check-online
(17:50:21) INFO:    [Connection]:       Pinging external endpoints
(17:50:21) INFO:    [Backend]:          DRM module staus {
  "oimompecagnajdejgnnjijobebaeigek": {
    "name": "Widevine Content Decryption Module",
    "status": "new",
    "version": "4.10.2710.0"
  }
}
(17:50:21) WARNING: [Backend]:          Failed to register protocol with OS.
(17:50:21) INFO:    [Connection]:       Connectivity: online
(17:50:21) INFO:    [Gog]:              Checking if login is valid
(17:50:21) INFO:    [Gog]:              Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(17:50:21) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=/home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(17:50:21) INFO:    [Gog]:              Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json --version
(17:50:21) INFO:    [Nile]:             Running command: NILE_CONFIG_PATH=/home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/nile_config /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/nile --version
(17:50:21) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(17:50:22) INFO:    [Backend]:          
System Information:
CPU: 12x AMD Ryzen 5 3600XT 6-Core Processor
Memory: 33.57 GB (used: 5.32 GB)
GPUs:
  GPU 0:
    Name: NVIDIA Corporation GA104 [GeForce RTX 3060 Ti Lite Hash Rate]
    IDs: D=2489 V=10de SD=4663 SV=3842
    Driver: nvidia
OS: Fedora Linux 39 (Workstation Edition) (linux)

The current system is not a Steam Deck
We are running inside a Flatpak container

Software Versions:
  Heroic: 2.11.0 Kumachi
  Legendary: 0.20.33 Undue Alarm
  gogdl: 0.7.3
  Nile: 1.0.0 Jonathan Joestar

(17:50:22) INFO:    [Backend]:          Loading Screen Ready
(17:50:22) INFO:    [Frontend]:         Refreshing undefined Library
(17:50:22) WARNING: [Backend]:          refresh not implemented on Sideload Library Manager
(17:50:22) INFO:    [Gog]:              Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(17:50:22) INFO:    [Legendary]:        Refreshing library...
(17:50:22) INFO:    [Legendary]:        Refreshing Epic Games...
(17:50:22) INFO:    [Legendary]:        Game list updated, got 217 games & DLCs
(17:50:22) INFO:    [Gog]:              Getting GOG library
(17:50:22) INFO:    [Gog]:              Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(17:50:22) INFO:    [Backend]:          Frontend Ready
(17:50:22) INFO:    [Backend]:          Checking for current version changelog
(17:50:22) INFO:    [Legendary]:        Running command: LEGENDARY_CONFIG_PATH=/home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(17:50:22) INFO:    [Gog]:              Saved username to config file
(17:50:23) INFO:    [Gog]:              Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json auth
(17:50:24) DEBUG:   [Backend]:          Unable to load Steam Libraries, libraryfolders.vdf not found
(17:50:24) INFO:    [Winetricks]:       Downloading Winetricks
(17:50:24) INFO:    [DXVKInstaller]:    Updating dxvk-nvapi to: dxvk-nvapi-v0.6.4
(17:50:25) INFO:    [Gog]:              Saved games data
(17:50:25) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(17:50:25) INFO:    [Gog]:              Found 0 game(s) to update
(17:50:25) INFO:    [DownloadManager]:  Downloader: Finished downloading https://github.com/jp7677/dxvk-nvapi/releases/download/v0.6.4/dxvk-nvapi-v0.6.4.tar.gz
(17:50:25) INFO:    [DownloadManager]:  Downloader: Finished downloading https://github.com/jp7677/dxvk-nvapi/releases/download/v0.6.4/dxvk-nvapi-v0.6.4.tar.gz
(17:50:25) INFO:    [DXVKInstaller]:    downloaded dxvk-nvapi
(17:50:25) INFO:    [DXVKInstaller]:    extracting dxvk-nvapi
(17:50:25) INFO:    [Backend]:          Extracting /home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/dxvk-nvapi/dxvk-nvapi-v0.6.4.tar.gz to /home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/dxvk-nvapi/dxvk-nvapi-v0.6.4 using native tar
(17:50:25) ERROR:   [Backend]:          Extracting Error: tar: /home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/dxvk-nvapi/dxvk-nvapi-v0.6.4: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now

(17:50:25) INFO:    [DXVKInstaller]:    dxvk-nvapi updated!
(17:50:25) INFO:    [Legendary]:        Checking for game updates: LEGENDARY_CONFIG_PATH=/home/helpvisa/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig/legendary /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(17:50:26) INFO:    [WineDownloader]:   Updating wine versions info
(17:50:26) INFO:    [WineDownloader]:   Fetching upstream information...
(17:50:27) INFO:    [Legendary]:        Found 0 games to update
(17:50:27) INFO:    [WineDownloader]:   wine versions updated
(17:50:27) INFO:    [Backend]:          Starting the Download Queue
(17:50:31) DEBUG:   [Backend]:          Unable to load Steam Libraries, libraryfolders.vdf not found
```

### Steps to reproduce

1. Go to the settings page of a given game
2. Attempt to enable 'Auto Install/Update DXVK-NVAPI on Prefix'
3. Watch an endless 'Please Wait...' animation loop

### Expected behavior

1. Go to settings page of a given game
2. Enable 'Auto Install/Update DXVK-NVAPI on Prefix'
3. DXVK-NVAPI is installed and activated, and the box becomes checked

### Screenshots

![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/89664809/a604bf7a-976d-4196-a239-44719126bc6e)
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/89664809/de0860b7-d721-4520-9390-698c7dc71000)

### Heroic Version

Latest Stable (Flatpak)

### System Information

- OS: Fedora Linux 39 x86_64 
- Kernel: 6.6.6-200.fc39.x86_64 
- GPU: NVIDIA GeForce RTX 3060 Ti
- CPU: AMD Ryzen 5 3600XT (12) @ 3.8GHz
- Running Gnome 45.2 with Xorg, not Wayland
- Nvidia Driver Version: 545.29.06 from RPM Fusion

**Comment by helpvisa** [comment]
Replying as I've done some additional testing after reading the log uploaded here again. I've uploaded a video of the behaviour occurring within the folder noted in the logs after copying manually the dxvk-nvapi tar into it, to see if I can't get Heroic to recognize it without having to download the file.

[Screencast from 2023-12-19 18-03-43.webm](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/89664809/d0de8115-4f93-4be7-bdac-9b81e8246919)

**Comment by helpvisa** [comment]
Manually creating the folder and extracting the tar into it seems to fix the issue and allows me to enable the auto-update / install option.

![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/89664809/52072ac8-50a8-4d5d-908b-aa7bf97228b0)

**Comment by FLagosV** [comment]
Had the same issue and I confirm what @helpvisa  did (to do the manual extraction of the dxvk tar) fixes the problem.

**Comment by jasonlevaillant** [comment]
Same problem with the .deb version of Heroic Games Launcher 2.11.0 and same fix.

**Comment by arielj** [comment]
This will be fixed in the next release by this PR https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/3347

until then you'll have to do the manual workaround from @helpvisa comment

**Comment by arielj** [comment]
Closing since this is fixed in 2.12.0
